### PR TITLE
CVE Protobuf: Avatica to 1.9 (protobuf upgrade) and dep management to PB 3.4.0

### PR DESCRIPTION
**Pull request by cjolivier01**
## Problem

https://snyk.io/vuln/maven:com.google.protobuf%3Aprotobuf-java

##### Does this solution apply anywhere else?
- [ ] yes
- [ ] no

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [ ] Manual tests

### CODE REVIEW

**Comment by cjolivier01** [comment]
> Shouldn't protobuf be covered by these changes in 11.0.x? https://github.com/confluentinc/kafka-connect-storage-common/blob/master/pom.xml#L178
> 
> Version pulled from https://github.com/confluentinc/common/blob/7.2.x/pom.xml#L90
> 
> Also how about we target 11.0.x?

I currently only know of a CVE against master, so that's the branch I am putting this against.  If you have one against 11.0.x, please assign to me.

**Comment by cjolivier01** [comment]
> Shouldn't protobuf be covered by these changes in 11.0.x? https://github.com/confluentinc/kafka-connect-storage-common/blob/master/pom.xml#L178
> 
> Version pulled from https://github.com/confluentinc/common/blob/7.2.x/pom.xml#L90
> 
> Also how about we target 11.0.x?

PB is shaded into avatica. it should shaded against a "safe" version closest to what it was meant to be built against.

**Comment by kpatelatwork** [comment]
@cjolivier01 I am also in need of this fix but in 10.0.x branch, can we please target that?

File: pom.xml [hunk 1]
```diff
@@ -90,6 +90,7 @@
         <json.smart.version>2.4.7</json.smart.version>
         <thrift.version>0.13.0</thrift.version>
         <tomcat.version>8.5.65</tomcat.version>
+        <protobuf.version>3.16.1</protobuf.version>
```
Comment (BlueEldur) [inline pom.xml:93]: can we make this 3.19.4 (latest)?
https://nvd.nist.gov/vuln/detail/CVE-2021-22569

**Review by BlueEldur — APPROVED** [review]
LGTM

**Pull request merged**